servers/physics/joints/generic_6dof_joint_sw.cpp

```diff
@@ -219,7 +219,7 @@ Generic6DOFJointSW::Generic6DOFJointSW(BodySW *rbA, BodySW *rbB, const Transform
 }
 
 void Generic6DOFJointSW::calculateAngleInfo() {
-	Basis relative_frame = m_calculatedTransformA.basis.inverse() * m_calculatedTransformB.basis;
```

Comment:
Same here, conventions have changed

Reply:
Basis is fixed to be R.S rather than S.R, but what does that have to do with this change?

Reply:
Also, below in this function, you may want to renormalize m_calculatedAxis below. I don't know how bad the drift due to errors is in general, but the current version of bullet [does it](https://github.com/bulletphysics/bullet3/blob/master/src/BulletDynamics/ConstraintSolver/btGeneric6DofConstraint.cpp#L353) apparently.